Common/Deployment/RemoteMonitoring.json

```diff
@@ -46,14 +46,14 @@
             "type": "string",
             "defaultValue": "http://aka.ms/azureiot/remotemonitoring/webjob"
         },
-        "sasStartBehavior": {
+        "asaStartBehavior": {
             "type": "string",
             "metadata": {
                 "description": "The start behavior for Stream Analytics jobs [LastStopTime | JobStartTime (default)]"
             },
             "defaultValue": "JobStartTime"
         },
-        "sasStartTime": {
+        "asaStartTime": {
```

Comment:
Would it make sense to also rename `sasStartTime` and `sasStartBehavior` in LocalMonitoring.json?